ring/testutils/testutils.go

```diff
@@ -0,0 +1,26 @@
+package testutils
+
+import (
+	"context"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+
+	"github.com/grafana/dskit/kv"
+	"github.com/grafana/dskit/ring"
+)
+
+// NumTokens determines the number of tokens owned by the specified
+// address
+func NumTokens(c kv.Client, name, ringKey string) int {
```

Comment:
Who is using it? I think none is using it in dskit. If confirmed, then I would not have it in dskit. Specific projects needing it can just keep their own copy.